### ISSUE TITLE
Dashboard widgets missing in Grav 1.7.0-rc

### ISSUE THREAD

**Issue by Jakub-Nagy** [getgrav/grav-plugin-admin#1913]
Hello,
I recently installed Grav 1.7.0-rc with the new Admin plugin. I have to say that it's amazing. But there is one little issue I have with the Dashboard. I have enabled all of the Dashboard Widgets:
![image](https://user-images.githubusercontent.com/37936430/85929096-5d772400-b8b2-11ea-80e5-708a6ab129d2.png)

But my dashboard looks like this:
![image](https://user-images.githubusercontent.com/37936430/85929103-7089f400-b8b2-11ea-8bd5-367773fe95b1.png)

What can cause this? I'm probably just missing some hidden setting. Thanks.

**Comment by ricardo118** [comment]
@Jakub-Nagy and your admin is also 1.10+? do you have any error showing up on the js console? I am on 1.7 too and i have my widgets going

**Comment by Jakub-Nagy** [comment]
@ricardo118 yes I have **"Grav v1.7.0-rc.12 - Admin v1.10.0-rc.12"**. My console shows just this, this shouldn't be important:
![image](https://user-images.githubusercontent.com/37936430/85932175-f5ccd300-b8c9-11ea-8f8e-0fcabc7d81e5.png)

Maybe there are some residual configs from the old Grav core, since I just pasted the new core over it.

**Comment by ricardo118** [comment]
this is a weird one, the widgets didnt get changed at all. would you mind reinstalling admin? delete the old one and download the latest release of github

**Comment by Jakub-Nagy** [comment]
@ricardo118 well, first I tried removing all of the admin config. Didn't help. Then I deleted the admin plugin and reinstalled the plugin from the **Grav Core + Admin plugin** bundle. Didn't help.

I downloaded the Admin plugin from the develop branch of this repository. I get a login window. But when I login, the effects see to be more severe:
![image](https://user-images.githubusercontent.com/37936430/85953073-3716be80-b96e-11ea-84ad-897ef9e10666.png)
